local-interchain/interchain/start.go

```diff
@@ -197,6 +200,19 @@ func StartChain(installDir, chainCfgFile string, ac *types.AppStartConfig) {
 		}
 	}
 
+	// ICS provider setup
+	if len(icsProviderPaths) > 0 {
+		logger.Info("ICS provider setup", zap.Any("icsProviderPaths", icsProviderPaths))
+
+		for ibcPath, chain := range icsProviderPaths {
+			if provider, ok := chain.(*cosmos.CosmosChain); ok {
+				if err := provider.FinishICSProviderSetup(ctx, relayer, eRep, ibcPath); err != nil {
+					log.Fatal("FinishICSProviderSetup", err)
```

Comment:
`local-ic start interchainsecurity`, manually tested and executed against a neutron testnet without issue :)

`local-ic interact localneutron-1 bin 'tx bank send acc0 neutron1hj5fveer5cjtn4wd6wstzugjfdxzl0xpznmsky 5untrn --keyring-backend=test --node=%RPC% --chain-id=%CHAIN_ID% --yes'`